src/main/java/org/example/lib/ORManagerImpl.java

```diff
@@ -25,8 +22,9 @@ public class ORManagerImpl implements ORManager {
     private static final String ID = " BIGINT PRIMARY KEY AUTO_INCREMENT";
     private static final String NAME = " VARCHAR(255) UNIQUE NOT NULL";
     private static final String DATE = " DATE NOT NULL";
-    private static final String INT = " INT NOT NULL";
+    private static final String INT = " BIGINT NOT NULL";
```

Comment:
This one "String INT ", should be changed as well, maybe like String LONG or similar.